recipes/hklpy2/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "hklpy2" %}
+{% set version = "0.0.16" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/hklpy2-{{ version }}.tar.gz
+  sha256: 159033cc0b81f9ac6f803ce50d39363cb704f64bb99d829630d13eaf42a30efc
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
```

You have some entrypoints there. You will should probably also add some command line testing for them in the testing section below.

```suggestion
  number: 0
  entry_points:
    - no_op = hklpy2.backends.no_op:NoOpSolver
    - hkl_soleil = hklpy2.backends.hkl_soleil:HklSolver
    - th_tth = hklpy2.backends.th_tth_q:ThTthSolver
```

---

Good catch!

---

done